aspnet/PdfViewer/Toolbar-Customization.md

```diff
@@ -1,14 +1,14 @@
 ---
 layout: post
-title: Toolbar customization of Syncfusion Essential ASP.NET PDF viewer.
-description: Toolbar customization of Syncfusion Essential ASP.NET PDF viewer.
+title: Toolbar customization in ASP.NET PDF viewer | Syncfusion
```

Comment:
Check H1, Title and description and update as ASP.NET Web Forms

Reply:
resolved